lib/amqp.js

```diff
@@ -145,16 +144,34 @@ AMQPConnection.prototype.sendToExchange = function sendToExchange(exchangeName,
     }
 };
 
+AMQPConnection.prototype.prepareMessageAndSendToExchange
```

Comment:
I'm sure we have some candidates in our team who love rewriting existing code in later stages. I don't see any business value in that and would like to avoid it at this stage.